myapp/app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :tasks
+  has_secure_password
+  validates :name, presence: true
```

Comment:
Can you implement validation?

Reply:
I added name verification

Reply:
Thank you!
I think it's better not to use spaces or symbols.

Reply:
To make it easier to manage, I've added name verification, which can only be entered in English and numbers.